src/ol_concourse/pipelines/open_edx/edx_platform_v2/grouped_docker_packer_pulumi_pipeline.py

```diff
@@ -0,0 +1,305 @@
+                                path="sh",
+                                args=[
+                                    "-xc",
+                                    f"""cp -r {release_name}-{deployment.deployment_name}-theme collected_themes/{deployment.deployment_name};
+                                        cp -rT {release_name}-{deployment.deployment_name}-edx-platform edx_platform;
+                                        echo "{release_name}-{deployment.deployment_name}-$(cat edx_platform/.git/short_ref)" > version/tag;""",  # noqa: E501
+                                ],
+                            ),
+                        ),
+                    ),
+                    container_build_task(
+                        inputs=[
+                            Input(name=edx_docker_code.name),
+                            Input(
+                                name=Identifier("collected_themes"),
+                                path=f"{edx_docker_code.name}/dockerfiles/openedx-edxapp/collected_themes",
+                            ),
+                            Input(
+                                name=Identifier("edx_platform"),
+                                path=f"{edx_docker_code.name}/dockerfiles/openedx-edxapp/edx_platform",
+                            ),
+                            Input(name=Identifier("version")),
+                        ],
+                        build_parameters={
+                            "CONTEXT": "ol-infrastructure-docker/dockerfiles/openedx-edxapp",  # noqa: E501
+                            "TARGET": "final",
+                            "BUILD_ARG_RELEASE_NAME": release_name,
+                            "BUILD_ARG_DEPLOYMENT_NAME": deployment.deployment_name,
+                        },
+                        build_args=[],
+                    ),
+                    PutStep(
+                        put=edx_registry_image_resource.name,
+                        params={
+                            "image": "image/image.tar",
+                            "additional_tags": "version/tag",
+                        },
+                    ),
+                ],
+            )
+
+            job_names.append(docker_build_job.name)
+
+            container_fragments.append(
+                PipelineFragment(
+                    resources=[
+                        edx_docker_code,
+                        edx_registry_image_resource,
+                        edx_ami_code,
+                        edx_pulumi_code,
+                        *theme_git_resources,
+                        *edx_platform_git_resources,
+                    ],
+                    jobs=[docker_build_job],
+                )
+            )
+
+            packer_fragments.append(
+                packer_jobs(
+                    dependencies=[
+                        GetStep(
+                            get=edx_registry_image_resource.name,
+                            trigger=False,
```

Comment:
A new docker image should trigger a new AMI build so that our CI stages are always up to date

Reply:
Yeah same with all of these. I didn't want jobs running w/o me explicitly asking. Will update.